src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.Analyzer.cs

```diff
@@ -54,21 +54,6 @@ protected override bool ReadOnlyFieldAllowed()
             return scope == null;
         }
 
-        protected override ITypeSymbol? GetSymbolType(SemanticModel semanticModel, ISymbol symbol)
-        {
-            var selectionOperation = semanticModel.GetOperation(SelectionResult.GetContainingScope());
-
-            // Check if null is possibly assigned to the symbol. If it is, leave nullable annotation as is, otherwise
-            // we can modify the annotation to be NotAnnotated to code that more likely matches the user's intent.
-            if (selectionOperation is not null &&
-                NullableHelpers.IsSymbolAssignedPossiblyNullValue(semanticModel, selectionOperation, symbol) == false)
-            {
-                return base.GetSymbolType(semanticModel, symbol)?.WithNullableAnnotation(NullableAnnotation.NotAnnotated);
-            }
-
-            return base.GetSymbolType(semanticModel, symbol);
-        }
```

Comment:
moved this to base class.  it will be a no-op in VB as we'll never see anything annotated at that layer.